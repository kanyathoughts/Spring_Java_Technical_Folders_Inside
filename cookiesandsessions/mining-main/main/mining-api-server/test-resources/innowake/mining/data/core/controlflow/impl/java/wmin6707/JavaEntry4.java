public class JavaEntry4 {

	public static void main(final String args[]) {

	}
}
