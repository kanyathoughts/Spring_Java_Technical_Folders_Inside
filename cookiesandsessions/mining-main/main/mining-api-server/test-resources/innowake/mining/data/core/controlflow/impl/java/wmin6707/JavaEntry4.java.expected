  #5 JavaMethodDeclaration, FROM={TestEntryPoint}, TO={TestReturnPoint}
   #16 JavaBlock
